ogr/services/pagure/issue.py

```diff
@@ -81,6 +90,17 @@ def labels(self) -> List[str]:
     def __str__(self) -> str:
         return "Pagure" + super().__str__()
 
+    def __update_info(self, new_title: str, new_description: str) -> None:
```

Comment:
What about making the arguments optional and have the `self.title` and `self.description` as default?